tests/oeedger8r/host/testdeepcopy.cpp

```diff
@@ -68,14 +476,40 @@ void test_deepcopy_edl_ecalls(oe_enclave_t* enclave)
     {
         auto s = init_struct<CountParamStruct>();
         OE_TEST(deepcopy_countparam(enclave, &s) == OE_OK);
+        // The members used by size/count attributes is expected to
+        // be consistent even if the callee modified them.
         OE_TEST(s.count == 7);
+        // The members not used by size/count attributes is expected to
+        // match the value set by the callee.
+        OE_TEST(s.size == 200);
         test_struct(s, s.count);
     }
 
+    {
+        auto s = init_struct<SizeStruct>();
+        OE_TEST(deepcopy_size(enclave, &s) == OE_OK);
+        test_struct(s, 2);
+    }
+
+    {
+        auto s = init_struct<CountSizeStruct>();
+        OE_TEST(deepcopy_countsize(enclave, &s) == OE_OK);
+        test_struct(s, 3);
+        // The members not used by size/count attributes is expected to
+        // match the value set by the callee.
+        OE_TEST(s.count == 5);
+        OE_TEST(s.size == 32);
+    }
+
     {
         auto s = init_struct<SizeParamStruct>();
         OE_TEST(deepcopy_sizeparam(enclave, &s) == OE_OK);
+        // The members used by size/count attributes is expected to
```

Comment:
nit: Fix grammar throughout "members ... is" -> "members ... are"

Reply:
done.